Makefile

```diff
@@ -32,10 +32,14 @@ test-integration: ## Run the integration testsuite.
 	composer test:integration --ansi
 
 test-matrix:
+	docker run --rm -t -v $$(pwd):/opt/graze/queue -w /opt/graze/queue php:5.5-cli \
```

Comment:
would be nice to update this Makefile to follow a bit more of: https://github.com/graze/dog-statsd/blob/master/Makefile style. Mainly to remove the need for composer to run the tests, we have a makefile for that.